src/test/java/software/amazon/awssdk/crt/test/CredentialsProviderTest.java

```diff
@@ -151,4 +157,41 @@ public Credentials getCredentials() {
             fail(ex.getMessage());
         }
     }
+
+    @Test
```

Comment:
Either in separate or combined tests, lets actually source credentials from your test files (or fail to source them).

Reply:
Good idea, I'll modify the `ValidProfile` test to check for real credentials.